src/modules/hyprland/workspaces.cpp

```diff
@@ -316,13 +317,13 @@ void Workspaces::onEvent(const std::string &ev) {
   std::string eventName(begin(ev), begin(ev) + ev.find_first_of('>'));
   std::string payload = ev.substr(eventName.size() + 2);
 
-  if (eventName == "workspace") {
+  if (eventName == "workspacev2") {
     onWorkspaceActivated(payload);
   } else if (eventName == "activespecial") {
     onSpecialWorkspaceActivated(payload);
-  } else if (eventName == "destroyworkspace") {
+  } else if (eventName == "destroyworkspacev2") {
     onWorkspaceDestroyed(payload);
-  } else if (eventName == "createworkspace") {
+  } else if (eventName == "createworkspacev2") {
     onWorkspaceCreated(payload);
   } else if (eventName == "focusedmon") {
     onMonitorFocused(payload);
```

Comment:
According to [the wiki](https://wiki.hyprland.org/IPC/), focusedmonv2, moveworkspacev2 and movewindowv2 are available. Perhaps you can upgrade those as well?